deps/debugging-tools.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+# vagrant-cookbook
+# Copyright (C) 2015 Pelagicore AB
```

Comment:
please do a search and replace on it in that case... 

As a sidenote, do you copyright stuff the year you create it, or the year you release it?